processing/src/main/java/io/druid/segment/DoubleColumnSelector.java

```diff
@@ -61,6 +61,9 @@ default long getLong()
   @Override
   default Double getObject()
   {
+    if (isNull()) {
```

Comment:
Make similar changes in FloatColumnSelector and LongColumnSelector

Reply:
done.